.env

```diff
@@ -19,8 +19,10 @@ APP_SECRET=6a7f172a8976663105d1801f2438235b
 ###< symfony/framework-bundle ###
 
 KEYCLOAK_HOME=http://localhost:52957
-KEYCLOAK_BASE=http://localhost:52957/auth/realms/master
```

Comment:
As in the "openid-configuration" which is in the realm->general section, I have got this URL and this is working fine.

Reply:
I have `http://localhost:52957/auth/realms/master` in this section. It's likely because you don't use Keycloak from the docker image shipped in the project, do you ? This stackoverflow thread seems confirm what I say: https://stackoverflow.com/questions/28658735/what-are-keycloaks-oauth2-openid-connect-endpoints. WDYT ?

Reply:
Yes. I'm using the java build of the Keycloak in one of my server instance. Not the docker image.